src/main/java/io/meeds/qa/ui/pages/page/factory/space/ManageSpacesPage.java

```diff
@@ -191,6 +191,11 @@ public class ManageSpacesPage extends GenericPage {
   @FindBy(xpath = "//*[contains(@class, 'v-dialog--active')]//button[contains(@class, 'btn-primary')]")
   private BaseElementFacade        removeConfirmationButton;
 
+  private BaseElementFacade getSpaceName(String fullName)
+  {
+    return findByXPathOrCSS(String.format("//div[contains(@class,'logoTitle') and contains(text(),'%s')]", fullName));
```

Comment:
IMO the xpath should be changed to be more consistent 
`//*[contains(@class,'UITopBarContainerItem')]//*[contains(@class,'logoTitle')]`